go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	github.com/google/flatbuffers v1.12.1 // indirect
 	github.com/gorilla/securecookie v1.1.1 // indirect
 	github.com/gorilla/sessions v1.2.1 // indirect
+	github.com/hashicorp/golang-lru/v2 v2.0.2 // indirect
```

Comment:
Shouldn't be `indirect`, please run `go mod tidy` to fix that.

Reply:
done